### ISSUE TITLE
Thunderscope: Simulator integration

### ISSUE THREAD

**Issue by akhilveeraghanta** [UBC-Thunderbots/Software#2410]
### Description of the task

Our simulator currently allows us to to:
* add robots 
* remove robots
* rotate robots
* move robots
* move ball
* set velocities
![features2](https://user-images.githubusercontent.com/31548846/149848454-e71fbb1b-ae54-43ed-b623-e18d21825a89.gif)

The new standalone er-force simulator (when its implemented/ready to be used) will have a protobuf interface for adding/removing/moving robots and moving the ball to a location with a velocity. 

The rest of the UI functionality needs to be implemented in thunderscope (up until we send the request to the er-force sim to update its state).

### Acceptance criteria

Each of these can be broken up into its own PR
- [ ] Figure out keyboard binding, and how to trigger a callback when a certain key is pressed. For example, if we are holding control and then click, we should be able to trigger a callback (teleport ball). Similar to R and click (to rotate)
- [ ] Figure out how to "select a robot". The interface to er-force sim will likely be based on robot id. So given the position the user clicked, we need to figure out which Robot ID was under the mouse (and what team)

If we have those two pieces (select robot and trigger callback based on keyboard + mouse combo) the rest of it is just populating the er-force sim "update your state" proto and sending it over to the standalone sim.

### Blocked By
Final testing is blocked until the standalone er-force sim is implemented. But, you should be able to figure out the keyboard + callback interactivity part and work out the python architecture for the callbacks

**Comment by williamckha** [comment]
Resolved by #3033
